js/lib/beautify-html.js

```diff
@@ -342,6 +347,16 @@
                         //no space after = or before >
                         this.space_or_wrap(content);
                         space = false;
+                        if (!first_attr && wrap_attributes === 'force' &&  input_char !== '/') {
+                            this.print_newline(true, content);
+                            this.print_indentation(content);
+                            for (var count = 0; count < wrap_attributes_indent_size; count++) {
+                                content.push(indent_character);
+                            }
+                        }
+                        if (content.filter(function (i) { return i === ' ';}).length === 1){
```

Comment:
This is terse and cool, but is not compatible with old browsers.  Could you do this as for loop?